src/lib/list/list.scss

```diff
@@ -131,14 +151,8 @@ $mat-list-item-inset-divider-offset: 72px;
     }
 
     &.mat-divider-inset {
-      left: $mat-list-item-inset-divider-offset;
-      width: calc(100% - #{$mat-list-item-inset-divider-offset});
       margin: 0;
-
-      [dir='rtl'] & {
-        left: auto;
-        right: $mat-list-item-inset-divider-offset;
-      }
+      position: absolute;
```

Comment:
The `position: absolute` is defined in the parent rule above, you don't have to add it to `.mat-divider-inset`. Also the nesting here isn't necessary.

Reply:
It was present to override a similar `position: static` in the `mat-card` styles. I added an `!important` tag instead with a note for why it's there. I can add the nested structure back if you prefer to not have `!important`

Reply:
In that case it can at least use a comment about why the nesting was necessary.

Reply:
Perfect, will add